server/asset/dcr/config.go

```diff
@@ -65,6 +65,8 @@ func loadConfig(configPath string, network dex.Network) (*config, error) {
 	// If no path provided, use default dcrd path.
 	if configPath == "" {
 		configPath = defaultConfigPath
+	} else {
+		defaultRPCCert = filepath.Join(filepath.Dir(configPath), "rpc.cert")
```

Comment:
If the config file doesn't specify `rpccert`, what should dcrdex do? Options are to look in either: (a) config file folder, or (b) any `appdata` set by the config file.  I think only (b) is correct in this case, although (a) seemed reasonable.

If the config file does not specify an `appdata`, that must mean either dcrd is using the default `appdata` OR the user started dcrd with `--appdata`.  If they started with `--appdata`, I don't think we can work around that. They will just have to edit the config file they provide to dcrdex.

So, I think the only "right" change to make now is: if no rpccert set, see if there is a custom `appdata` set (regardless of where `configPath` comes from), and reset `defaultRPCCert` to look there.  We could argue that if neither `rpccert` or `appdata` are set, they it should adjust as with this diff in the `configPath`, but that's not necessarily correct, and the resolution is simply to set one of those options in the config file.